### DISCUSSION TITLE
Why limit the JSON field outputs to 80 characters?

### DISCUSSION THREAD

**Discussion by tylere** [opengeos/aws-open-data#2] [Q&A] [answered]
Why are many of the fields in the JSON output [being limited to 80 characters](https://github.com/opengeos/aws-open-data/blob/master/aws_open_datasets.py#L11)? This causes useful information to disappear from the JSON summaries.

**Comment by giswqs** [comment]
The intention was to create a clean CSV file with dataset ids that can be passed into other programs. Many columns with long descriptions were not essential compared to dataset id, data range, spatial extent, etc.

But I agree with you, it leads to some missing information in the JSON file. While I think it is ok to limit the char length for the resulting csv file, we should keep the complete information in the JSON file. I will update it tomorrow.

**Comment by tylere** [comment]
Similarly, it might be better to preserve separate tags in the JSON output, before joining them for the clean CSV output. This would avoid having JSON users need to split the tags before using them.

**Reply by giswqs** [reply]
Good suggestion. Let me work on it.

**Reply by giswqs** [reply, marked as answer]
Sorry for the delay. I was traveling the past two days. This issue has been fixed in #3. Check out the updated JSON file.
